CODEOWNERS

```diff
@@ -11,7 +11,7 @@ packages/app-bridge/src/errors/ValidationError.ts @Frontify/integration
 ## App Bridge Helpers
```

Comment:
Github complains about an error in codeowners - on line 2 `@Frontify/ecosystem` should now be `@Frontify/api-and-sdk`